---
title: "Single-cell transcriptional profiling reveals cellular diversity and intercommunication in the mouse heart"
date: 2021-07-30T22:09-05:00
#November 23, 2019 at 22:09 EST
excerpt: Skelly et al., 2018, Cell Reports 22, 600–610
January 16, 2018 ª 2017 The Author(s).
https://doi.org/10.1016/j.celrep.2017.12.072
categories:
  - Daily Paper
---
## Introduction
Essentially they are using single-cell RNA sequencing on mice to understand gene expression of all cell types in the heart except for cardiomyocytes. In scRNA-seq, you isolate RNA from a single cell, use reverse transcription to convert it into complementary DNA, and then amplify that DNA either through the use of PCR or in vitro transcription. With that amplified DNA, you can then generate a sequencing library and sequence it. This method is advantageous because it can often reveal rare cell types in a population. However, one caveat is that it is impossible to gather 100% information on all RNA in each of the cells due to the very small amount present. Overall, thoguh, these experiments allow us to understand the structure and function of the cardiac cell makeup. 

## Results
**Figure 1:** "scRNA-Seq Analysis of Cardiac Non-myocytes Reveals Major Cardiac Cell Types
<ol type="a">
  <li>Shows the methodology of selecting cells for scRNA-seq. They reduce the percentage of endothelial cells to approx. 10% of total non-myocytes because they did not want to oversample endothelial cells (largest cardiac cell population and is mostly made of miscrovascular endothelial cells).
  | Endothelial cells | |
  | Fibroblasts | |
  |Granulocytes | |
  </li>
  <li>Through dimensionality reducation and unsupervised cell clustering, 12 distinct cell clusters were revealed.</li>
  <li>Milk</li>
</ol>

## Discussion

## References
[^1]: https://www.frontiersin.org/articles/10.3389/fnmol.2017.00177/full
[^2]: https://www.mayoclinic.org/diseases-conditions/cardiomyopathy/symptoms-causes/syc-20370709
[^3]: 
[^4]: 
[^5]: 
[^6]: 
[^7]: 
[^8]: 
[^9]: 
[^10]: 